### PR TITLE
Outline and objectives

### PR DESCRIPTION
**Pull request by yochannah**
Moving in the outline and objectives

### CODE REVIEW

File: lessons/lesson2.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Lesson 2: WHY: Benefits and Challenges of Responsible Open Science - Why does it matter?
+
+## Introduction
+
+## Benefits of Open Science
+
+### Quality of research
+
+### Quality and diversity of scholarly communications
```
Comment (Ismael-KG) [inline lessons/lesson2.md:9]: I am unsure the content of this subsection adequately covers diversity. This is (as all others) just a suggestion!

```suggestion
### Quality of scholarly communications
```

File: lessons/lesson3.md [hunk 2]
```diff
@@ -1,27 +1,35 @@
-# Lesson 3: Anatomy of a module
+# Lesson 3: WHO: Stakeholders of Open Science: Who practices Responsible Open Science and for whom?
```
Comment (Ismael-KG) [inline lessons/lesson3.md:1]:
```suggestion
# Lesson 3. Stakeholders of Open Science: Who practices Responsible Open Science and for whom?
```

File: lessons/lesson3.md [hunk 3]
```diff
@@ -1,27 +1,35 @@
...
-- 8 lessons of 15 minutes reading time each
-- five 3-min videos
-- 15 minutes in estimated time for recall activities
+## How each group contributes to Open Science
 
-## Video
+### Case scenarios
 
-The use of video can be sparse, but it is effective for giving the pig picture, creating a human connection with learners, and helping to keep them on track.
+#### Case Scenario #1:  Trend: Public  —> Policy-makers
```
Comment (Ismael-KG) [inline lessons/lesson3.md:17]:
```suggestion
#### Case Scenario #1:  Trend: Public —> Policy-makers
```

File: lessons/lesson4.md [hunk 4]
```diff
@@ -1,34 +1,23 @@
-# Lesson 4: Self checks, assessments, and activities
+# Lesson 4: WHERE: Impact of Open Science on academia,  communities and society as a whole: Where open science happens
```
Comment (Ismael-KG) [inline lessons/lesson4.md:1]:
```suggestion
# Lesson 4. Impact of Open Science on academia, communities and society as a whole: Where open science happens
```

File: lessons/lesson4.md [hunk 5]
```diff
@@ -1,34 +1,23 @@
...
+## Introduction
 
-Assessments can also be multiple-choice questions, but this time they are “graded” in an online course. Each module needs a few graded assessments, aligned with the learning objectives and the content, as presented. The modules will award a badge to learners that complete the assessments above a threshold score.
+## Legal frameworks that affect responsible open science
 
-Activities can be guided mini-projects for the learners to carry out on their own, perhaps after finishing a module, or a set of tasks to complete while guided by an instructor at an in-person event. We strongly encourage course creators to design a handful of activities for each module. You can explain in writing the activities for the learners, and you can provide a brief description for instructors to have some ideas about how to facilitate the activities in person.
+### Data protection, privacy, and data sovereignty
 
-Two principles of pedagogy are relevant to this guidance:
+### European case: General Data Protection regulation
```
Comment (Ismael-KG) [inline lessons/lesson4.md:9]:
```suggestion
### European case: the General Data Protection Regulation
```

File: lessons/lesson4.md [hunk 6]
```diff
@@ -1,34 +1,23 @@
...
-2. **Constructive alignment** — combines the learning theory of constructivism (that learners construct their knowledge or create meaning from active participation), and the design principle of alignment (that content and assessments should be aligned with learning objectives). Instructional designers often champion the idea of starting with defining learning objectives (known as backward design), but in practice course creation is often an iterative process. Do not become obsessed with learning objectives, but do keep in mind the principle of alignment.
+### South African case: Protection of Personal Information Act (POPI Act) and Open Science
 
-:::{seealso}
-Read more about constructive alignment, especially as it concerns accessible education, in [chapter 3](https://flexforward.pressbooks.com/chapter/constructive-alignment/) of the online book "Forward with FLEXibility,"  by Alise de Bie and Kate Brown (2017).
-:::
+### United States case
```
Comment (Ismael-KG) [inline lessons/lesson4.md:13]: Adding something after a semicolon to follow the pattern of the EU and South African cases above.

```suggestion
### United States case: state legislation
```

File: lessons/lesson5.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
+# Lesson 5: HOW: Open Science resources & tools: How to get started in responsible Open Science (doing open science responsibly)
```
Comment (Ismael-KG) [inline lessons/lesson5.md:1]:
```suggestion
# Lesson 5: Open Science resources & tools: How to get started in responsible Open Science
```

**Review by Ismael-KG — CHANGES_REQUESTED** [review]
Hoping my review helps. The two main comments:
* Add blank spaces between titles in lesson one to be consistent with the remaining lessons (or remove spaces in the remaining lessons!); and
* Drop "What, Why, How, etc." from the start of lesson titles to make them more legible (unless this will be done across all the modules).

File: lessons/lesson1.md [hunk 8]
```diff
@@ -1,43 +1,9 @@
...
-
-To insert a bibliography into your page, you will use the `{bibliography}` directive. For example, if the references for your module are stored in `references.bib`, you indicate this file name in the `_config.yml` and
-then insert the bibliography with:
-
-````
-```{bibliography}
-```
-````
\ No newline at end of file
+# Lesson 1: WHAT: Intro to Open Science: What is it and what does it promote?
```
[the remark this answers is not in the document]
Comment (yochannah) [reply lessons/lesson1.md:1]: This agrees with something I'd been wondering about - so yes, this is a good idea. Thanks!

File: lessons/lesson1.md [hunk 9]
```diff
@@ -1,43 +1,9 @@
...
+## Introduction
+## Context and Definition
+## Definitions of Open Science and Responsible Open Science
+## Open Science aspects
+## There is no one ethos
+## Performing open science responsibly
+## Summary
+## Further Reading
```
[the remark this answers is not in the document]
Comment (yochannah) [reply lessons/lesson1.md]: addressed in most recent push!

**Review by aosman12 — APPROVED** [review]
Looks Good!

**Review by aosman12 — APPROVED** [review]

**Pull request merged**